samples/snippets/csharp/VS_Snippets_CLR_System/system.String.Format/cs/starting1.cs

```diff
@@ -41,18 +41,18 @@ private static void Snippet33()
        // <Snippet33>
        int[] years = { 2013, 2014, 2015 };
        int[] population = { 1025632, 1105967, 1148203 };
-       String s = String.Format("{0,6} {1,15}\n\n", "Year", "Population");
-       for(int index = 0; index < years.Length; index++)
-          s += String.Format("{0,6} {1,15:N0}\n",
-                             years[index], population[index]);
+       StringBuilder myStringBuilder = new StringBuilder();
```

Comment:
this is a nit, but it would be good to name the variable something other than my(whatever). Perhaps simply `sb`, or `strBuilder`?